### ISSUE TITLE
Training command

### ISSUE THREAD

**Issue by jingfengguo-ai** [Talegqz/neural_novel_actor#3]
Thanks for your great work, I wanted to ask if there is any command to train on the ZJU dataset

**Comment by Talegqz** [comment]
Sorry, we haven't sorted out the training code yet.

**Comment by jingfengguo-ai** [comment]
> Sorry, we haven't sorted out the training code yet.

Is there any plan to open source the training code？

**Comment by Talegqz** [comment]
Yes, we do have a plan. But we are rushing to meet a deadline, if you have an urgent need, you can email us.
